CodeNow/runnable-angular
# SAN-2478 Fix empty port state

Fixed bug where removing the ports wouldn't remove the <user-specified-ports> section :)

To test:
1. Remove all the ports
2. Build
3. Notice it succeeded instead of crashed
